h1st/requirements.txt

```diff
@@ -1,28 +1,46 @@
-# data management & processing
-NumPy >= 1.18.4, < 1.19
-Pandas >= 1.0.4, < 1.2
-PyArrow >= 0.17.1
-cloudpickle == 1.3.0
-Ray == 0.8.6
-# TSFresh
-
-Click >= 7.1.1
-Colored >= 1.4.2
+# DATA MANAGEMENT & PROCESSING
+CloudPickle >= 1.6.0
+NumPy >= 1.19.5
+Pandas >= 1.2.1
+PyArrow >= 2.0.0
+# Ray[tune] >= 1.1.0
+S3FS >= 0.4.2, < 0.5
+
+
+# GRAPH PROCESSING
+GraphViz >= 0.16
+
+
+# MACHINE LEARNING
+SciKit-Learn >= 0.24.1
+LIME >= 0.2.0.1
+SHAP >= 0.38.1
+
+# HYPER-PARAMETER TUNING
+ConfigSpace >= 0.4.17
+Hpbandster >= 0.7.4
+Bayesian-Optimization >= 1.2.0   # for BO
+Tabulate >= 0.8.7
 
-# graph
-GraphViz == 0.14
 
-# Machine Learning
-SciKit-Learn
-boto3
-s3fs==0.4.2
-ulid-py==0.1.0
-lime==0.2.0.1
-shap==0.35.0
+# FUZZY LOGIC
+SciKit-Fuzzy >= 0.4.2
 
-# Fuzzy Logic
-scikit-fuzzy==0.4.2
 
-# IPython packages
-ipython==7.16.1
-ipython-genutils==0.2.0
+# IPython
+IPython >= 7.19.0
```

Comment:
is there any package not required for production @TheVinhLuong102. If yes, please move it to dev requirements.txt